common/src/range_key.rs

```diff
@@ -108,6 +108,19 @@ impl U64BlobIdClosedInterval {
   pub fn end(&self) -> UniqueU64BlobId {
     self.right
   }
+
+  pub fn iter(&self) -> impl Iterator<Item = UniqueU64BlobId> + '_ {
```

Comment:
This one is clear, but what you will truly need is an iterator over multiple key ranges, i.e. over the epoch, right?

Reply:
Not sure yet. Maybe, will depend on how this interface implementation will be used. Maybe even interface will be changed.